src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTag.cs

```diff
@@ -68,63 +72,81 @@ internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
         /// <param name="textView">The view of the editor</param>
         /// <param name="span">The span that has the location of the hint</param>
         /// <param name="key">The symbolkey associated with each parameter</param>
-        public static InlineParameterNameHintsTag Create(string text, TextFormattingRunProperties format,
-                                                         IWpfTextView textView, SnapshotSpan span, SymbolKey key,
-                                                         InlineParameterNameHintsTaggerProvider taggerProvider)
+        public static InlineHintsTag Create(
+            ImmutableArray<SymbolDisplayPart> parts,
+            TextFormattingRunProperties format,
+            IWpfTextView textView,
+            SnapshotSpan span,
+            SymbolKey? key,
+            InlineHintsTaggerProvider taggerProvider,
+            IClassificationFormatMap formatMap,
+            bool classify)
         {
-            return new InlineParameterNameHintsTag(CreateElement(text, textView, format), textView,
-                                                   span, key, taggerProvider);
+            return new InlineHintsTag(
+                CreateElement(parts, textView, span, format, formatMap, taggerProvider.TypeMap, classify),
+                textView, span, key, taggerProvider);
         }
 
         public async Task<IReadOnlyCollection<object>> CreateDescriptionAsync(CancellationToken cancellationToken)
         {
-            var document = _textView.TextBuffer.CurrentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
```

Comment:
this method added a null check and changed indentation.  it's otherwise unchanged.